### ISSUE TITLE
admin site breaks. TemplateSyntaxError at /admin/

### ISSUE THREAD

**Issue by albertorp** [wrabit/django-cotton#68]
Hi,

I've started testing cotton and I like it very much. I've found an issue when using Django-cotton, the admin site breaks and I get the following exception:

TemplateSyntaxError at /admin/
Invalid block tag on line 96: 'endblock', expected 'elif', 'else' or 'endif'. Did you forget to register or load this tag?

I have reproduced the error with a very simple django project created with the "startproject" command and then moving directly to the QuickStart section of the django-cotton section and updating the settings. 

If I then go to the admin site, I will get the error. This disappears if I comment out the line 'django_cotton.cotton_loader.Loader' from the loaders section in settings.py

Hope this helps. 

Here is the full output of the error I get

```
Environment:


Request Method: GET
Request URL: http://127.0.0.1:8000/admin/

Django Version: 5.0.7
Python Version: 3.12.1
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django_cotton']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware']


Template error:
In template /Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/contrib/admin/templates/admin/base.html, error at line 96
   Invalid block tag on line 96: 'endblock', expected 'elif', 'else' or 'endif'. Did you forget to register or load this tag?
   86 :           {% include "admin/nav_sidebar.html" %}
   87 :         {% endblock %}
   88 :       {% endif %}
   89 :       <main id="content-start" class="content" tabindex="-1">
   90 :         {% block messages %}
   91 :           {% if messages %}
   92 :             <ul class="messagelist">{% for message in messages %}
   93 :               <li{% if message.tags %} class="{{ message.tags }}" {% endif %}>{{ message|capfirst }}
   94 :             {% endfor %}</li{% if message.tags %}></ul>
   95 :           {% endif %}
   96 :          {% endblock messages %} 
   97 :         <!-- Content -->
   98 : <div id="content" class="{% block coltype %}colM{% endblock %}">
   99 :           {% block pretitle %}{% endblock %}
   100 :           {% block content_title %}{% if title %}<h1>{{ title }}</h1>{% endif %}{% endblock %}
   101 :           {% block content_subtitle %}{% if subtitle %}<h2>{{ subtitle }}</h2>{% endif %}{% endblock %}
   102 :           {% block content %}
   103 :             {% block object-tools %}{% endblock %}
   104 :             {{ content }}
   105 :           {% endblock %}
   106 :           {% block sidebar %}{% endblock %}


Traceback (most recent call last):
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 502, in parse
    compile_func = self.tags[command]
                   ^^^^^^^^^^^^^^^^^^

During handling of the above exception ('endblock'), another exception occurred:
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/response.py", line 92, in rendered_content
    return template.render(context, self._request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
           
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 510, in parse
    raise self.error(token, e)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 510, in parse
    raise self.error(token, e)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/defaulttags.py", line 961, in do_if
    nodelist = parser.parse(("elif", "else", "endif"))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/alberto/Development/django/tests/cotton/.venv/lib/python3.12/site-packages/django/template/base.py", line 555, in invalid_block_tag
    raise self.error(
    ^

Exception Type: TemplateSyntaxError at /admin/
Exception Value: Invalid block tag on line 96: 'endblock', expected 'elif', 'else' or 'endif'. Did you forget to register or load this tag?

```

**Comment by wrabit** [comment]
@albertorp thanks for reporting the issue. I have a PR lined up, but before I merge I want to ensure enough test coverage is around it. I'll get back to it when I have some more time soon.

https://github.com/wrabit/django-cotton/pull/69

**Comment by wrabit** [comment]
@albertorp fixed and released in [0.9.23](https://github.com/wrabit/django-cotton/releases/tag/v0.9.23)

**Comment by albertorp** [comment]
Great! It's working after the upgrade. Thanks a lot!
